### PR TITLE
[Pred] Adds a no block alternative to combistick

### PR DESCRIPTION
**Pull request by JayTacos**
# About the pull request

This pr adds in the War axe, as an alternative to the combistick. This weapon has no block chance (as seen below), and is intended to be a sidegrade from the combi. In return, it is a one handed weapon. Thus, the combi is left as is, and will continue to be an option for future hunts.

# Explain why it's good for the game

People really dislike the 30% block chance that comes associated with the combi. Landing shots or slashes against an already fast, tanky antagonist but having the game tell you "no" is not fun or intuitive. This PR is intended for the people who enjoy the throwing and tethered aspect of the combi, but don't want to guarantee their victory against anything short of a marine mob or a hive offensive.

# Testing Photographs and Procedure
Code was tested and ensured that it worked. Combi functions as normal, and war axe functions as a correct copy of it (you are not able to "fold down" the war axe, like you are with the combi).
Sprites are courtesy of Thwomper; huge shoutout to you for helping me through my first project.

image

<details>
<summary>Screenshots & Videos</summary>

How to choose it from the claim gear verb

https://github.com/user-attachments/assets/2fca01ee-b88b-4756-9bb4-6d8800d021c7

Functions like a combi. Yank Combi-stick is now called Yank Weapon, but still uses the internal guts of combi keybinding (this will not break your keybinds)

https://github.com/user-attachments/assets/e2d8f307-e383-4405-aeb3-bc077feb3f1b

Reference of Combistick blocking

https://github.com/user-attachments/assets/1a666d8f-e943-4153-8f33-9053bd033bfc

Showcase of War axe having 0 block chance

https://github.com/user-attachments/assets/8f9612b4-3925-4c68-840e-dca8467c85bc

Thralls are still able to pick the combi. They do not have an option for the alt version (similar to the other alt weapons)

https://github.com/user-attachments/assets/f840f866-fd85-4e1c-954f-cd404c73ec06

</details>

# Changelog

:cl:
add: Added a varient of combistick that has no block chance.
/:cl:

### CODE REVIEW

File: code/modules/cm_preds/yaut_weapons.dm [hunk 1]
```diff
@@ -230,7 +230,7 @@
 	item_state = "scythe_double"
 
 //Combistick
-/obj/item/weapon/yautja/combistick
+/obj/item/weapon/yautja/combistick/real
```
Comment (realforest2001) [inline code/modules/cm_preds/yaut_weapons.dm:233]:
```suggestion
/obj/item/weapon/yautja/chained/combistick
```
Repath all this to /obj/item/weapon/yautja/chained/ so the typepath is indicative of what the weapon actually is, as the axe is not a combistick at all, it's just copying the code to be attached to the pred by a chain.

**Review by realforest2001 — CHANGES_REQUESTED** [review]
Above

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/modules/cm_preds/yaut_weapons.dm (hunk 1 above)
Comment (JayTacos) [reply code/modules/cm_preds/yaut_weapons.dm:233]: Completely agree. working on it now

Comment (JayTacos) [reply code/modules/cm_preds/yaut_weapons.dm]: repathed it all, retested and I've noticed no issues. Thanks forest

**Review by realforest2001 — APPROVED** [review]

**Review by Blundir — APPROVED** [review]

**Comment by Blundir** [comment]
doesn't intervene with what i do

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request merged**